### PR TITLE
adding option for s3-proxy

### PR DESCRIPTION
**Pull request by isichei**

### CODE REVIEW

File: deploy.schema.json [hunk 1]
```diff
@@ -129,6 +129,11 @@
               "items": {
                 "type": "string"
               }
+            },
+            "health_check": {
+              "type": "string",
+              "description": "Path http path for livenessProbe in webapp deployment",
```
Comment (r4vi) [inline deploy.schema.json]: guessing this is used for both livenessProbe and readinessProbe?

**Comment by r4vi** [comment]
could you update the package version too: https://github.com/moj-analytical-services/deploy.json/blob/master/package.json#L3 (not actually used for anything except consistency)

File: deploy.schema.json (hunk 1 above)
Comment (isichei) [reply deploy.schema.json]: No just livenessProbe... Is that weird / wrong?

Comment (r4vi) [reply deploy.schema.json]: It's fine in this case but maybe a bit confusing for the user if they're
doing something different. I'd make this value change both

**Pull request merged**